### PR TITLE
Updates test/e2e/framework/gpu_util.go to use 1.9 GPUDevicePluginDSYAML.

### PR DESCRIPTION
**Pull request by jiayingz**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note

```

### CODE REVIEW

File: test/e2e/framework/gpu_util.go [hunk 1]
```diff
@@ -32,7 +32,7 @@ const (
 	// TODO: Parametrize it by making it a feature in TestFramework.
 	// so we can override the daemonset in other setups (non COS).
 	// GPUDevicePluginDSYAML is the official Google Device Plugin Daemonset NVIDIA GPU manifest for GKE
-	GPUDevicePluginDSYAML = "https://raw.githubusercontent.com/kubernetes/kubernetes/master/cluster/addons/device-plugins/nvidia-gpu/daemonset.yaml"
```
Comment (vishh) [inline test/e2e/framework/gpu_util.go]: I think we want to test both stable and master branches. `stable` is what we want users to use, but we want to make changes first to `master` branch.

Comment (jiayingz) [reply test/e2e/framework/gpu_util.go]: This is in 1.9 release branch. On head, we are testing against master.

**Review by rohitagarwal003 — APPROVED** [review]
/lgtm

**Comment by mindprince** [comment]
/assign @soltysh 
Can you approve this as well? Thanks!

**Comment by soltysh** [comment]
This needs to be approved by 1.9 release manager, @mbohlool

**Pull request merged**